dynawo/sources/API/JOB/JOBConstraintsEntryImpl.h

```diff
@@ -61,6 +61,9 @@ class ConstraintsEntry::Impl : public ConstraintsEntry {
    */
   std::string getExportMode() const;
 
+  /// @copydoc ConstraintsEntry::clone()
```

Comment:
Please keep the same coding style as above (same for all)

Reply:
Which coding style ?

Reply:
for the doxygen in this /** is used and not ///